<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <widget class="QWidget" name="main">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/assests/whiteIcon/home.svg</normaloff>:/icons/assests/whiteIcon/home.svg</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
      <widget class="QWidget" name="clients">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/assests/whiteIcon/users.svg</normaloff>:/icons/assests/whiteIcon/users.svg</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QWidget" name="AddClient" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>831</width>
          <height>181</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#AddClient{
background-color : #00ebc7;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Name :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="clientName">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Phone :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="clientPhone">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Email :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="clientEmail">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Address :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="clientAddress">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addClient">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>38</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/icons/assests/whiteIcon/check.svg</normaloff>:/icons/assests/whiteIcon/check.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="clientInfo" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>831</width>
          <height>301</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>821</width>
           <height>231</height>
          </rect>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/assests/whiteIcon/user.svg</normaloff>:/icons/assests/whiteIcon/user.svg</iconset>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Phone</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/assests/whiteIcon/phone.svg</normaloff>:/icons/assests/whiteIcon/phone.svg</iconset>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Email</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/assests/whiteIcon/mail.svg</normaloff>:/icons/assests/whiteIcon/mail.svg</iconset>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Address</string>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/assests/whiteIcon/map-pin.svg</normaloff>:/icons/assests/whiteIcon/map-pin.svg</iconset>
          </property>
         </column>
        </widget>
        <widget class="QPushButton" name="load">
         <property name="geometry">
          <rect>
           <x>374</x>
           <y>262</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="product">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/assests/whiteIcon/archive.svg</normaloff>:/icons/assests/whiteIcon/archive.svg</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
      <widget class="QWidget" name="settings">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/icons/assests/whiteIcon/settings.svg</normaloff>:/icons/assests/whiteIcon/settings.svg</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
